src/components/molecules/UiStepper/UiStepper.stories.ts

```diff
@@ -73,6 +80,40 @@ const meta = {
       control: 'object',
     },
   },
+  decorators: [ (story, { id }) => {
```

Comment:
import { useArgs } from '@storybook/preview-api'; can be used only in Stories function than I add new decorator to solve this issue